vagrant-libvirt.gemspec

```diff
@@ -17,9 +17,9 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.version       = VagrantPlugins::ProviderLibvirt.get_version
 
-  s.add_development_dependency "rspec-core", "~> 3.5"
-  s.add_development_dependency "rspec-expectations", "~> 3.5"
-  s.add_development_dependency "rspec-mocks", "~> 3.5"
+  s.add_development_dependency "rspec-core", "~> 3.10"
+  s.add_development_dependency "rspec-expectations", "~> 3.10"
+  s.add_development_dependency "rspec-mocks", "~> 3.10"
```

Comment:
Maybe we could just lift the constraints and rely on what is listed in vagrant? e.g. either remove the version completely or use ">= 3.5", I don't think we're really tied that tightly, more likely it was to reduce risk of surprises.

Reply:
Sounds like a more reasonable approach, let's see what the CI says about that 😉